Heat-Bundle/pom.xml

```diff
@@ -2,34 +2,28 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>hcom.funcTest</groupId>
+        <groupId>com.hotels</groupId>
         <artifactId>Heat-Core</artifactId>
-        <version>1.0</version>
+        <version>1.0.0</version>
```

Comment:
I would have all the versions set to 1.0.0-SNAPSHOT until we're ready to do a release. Should be easy to change them all in one go using the versions plugin.